default/stringtables/ru.txt

```diff
@@ -13352,31 +13521,46 @@ GOOD_PLANETARY_DEFENSE_DESC
 +	Крепкая планетарная [[metertype METER_DEFENSE]]: +5
 
 BROAD_EP_DESC
-++	Хорошая приспособляемость, может обитать на большем количестве типов планет
+++	Хорошая приспособляемость: может обитать на большем количестве типов планет.
 
 NARROW_EP_DESC
--	Плохая приспособляемость, может обитать на меньшем количестве типов планет
+-	Плохая приспособляемость: может обитать на меньшем количестве типов планет.
 
 FAST_COLONIZATION_DESC
 +	Быстрая колонизация: -25% к времени основания колонии
 
 SLOW_COLONIZATION_DESC
 −	Медленная колонизация: +20% к времени основания колонии
 
+NO_FUEL_DESC
+−−−	Корабли имеют нулевое [[metertype METER_FUEL]]
+
 BAD_FUEL_DESC
 −	Плохой максимум [[metertype METER_FUEL]]: - 1
 
+AVERAGE_FUEL_DESC
+	Средний максимум [[metertype METER_FUEL]]: +0
+	
+GOOD_FUEL_DESC
++	Хороший максимум [[metertype METER_FUEL]]: +1
+
+GREAT_FUEL_DESC
+++	Отличный максимум [[metertype METER_FUEL]]: +2
+
+ULTIMATE_FUEL_DESC
++++	Беспрецедентный максимум [[metertype METER_FUEL]]: +3
+
 GREAT_ASTEROID_INDUSTRY_DESC
-+	Богатые ресурсами шахты: + 0.2 [[metertype METER_INDUSTRY]] к населению, когда промышленность сосредоточена в системах с принадлежащими астероидными поясами
++	Хорошие астероидные добытчики: + 0.2 [[metertype METER_INDUSTRY]] за единицу населения, когда планета с фокусом на промышленность расположена рядом с астероидными поясами.
 
 LIGHT_SENSITIVE_DESC
-−	Светочувствительный: Население уменьшается в системах с [[STAR_BLUE]] и в меньшей степени с [[STAR_WHITE]] звездами
+−	Светочувствительность: Население уменьшается в системах с [[STAR_BLUE]] и в меньшей степени с [[STAR_WHITE]] звездами.
 
 TELEPATHIC_DETECTION_DESC
-+	Телепатическое обнаружение: может ощущать близлежащие населенные планеты
++	Телепатическое обнаружение: может ощущать близлежащие населенные планеты.
 
 COMMUNAL_VISION_DESC
-'''	Общая концепция: разделяет обзор в рамках одинаковых видов'''
+	Общая концепция: разделяет обзор в рамках одинаковых видов.
```

Comment:
Кавычки подчёркивают наличие отступа в начале значения.  Если отступ нужен, то кавычки стоит оставить

Reply:
Хз. Думаю, лучше тогда оставить :)

Reply:
Проверила англ. вариант, там тоже нет. Похоже, можно всё-таки убрать. :)

Reply:
Sorry, that's another time I done something strange in Git. (think, another uncorrect update from upstream).
I using instructions giving by Dilvish once again, so that's why there are few strange commits on log.

Reply:
Тогда давай уберём.